pbiviz.json

```diff
@@ -4,12 +4,12 @@
     "displayName": "ChicletSlicer",
     "guid": "ChicletSlicer1448559807354",
     "visualClassName": "ChicletSlicer",
-    "version": "2.0.8",
+    "version": "1.3.0",
```

Comment:
The latest available version of Chiclet Slcier is [1.2.14](https://app.powerbi.com/visuals/show/ChicletSlicer1448559807354). So, I decided to use 1.3.0 instead of 2.x.x for this update in order not to increase the major number (aka Semver).